systemtest/src/main/java/io/strimzi/systemtest/utils/kafkaUtils/KafkaUtils.java

```diff
@@ -394,16 +393,6 @@ public static void waitForKafkaDeletion(String namespaceName, String kafkaCluste
             () -> LOGGER.info(KafkaResource.kafkaClient().inNamespace(namespaceName).withName(kafkaClusterName).get()));
     }
 
-    public static String getKafkaTlsListenerCaCertName(String namespaceName, String clusterName, String listenerName) {
```

Comment:
why this removal?

Reply:
It was probably an unused method which I removed together with the rest. When removing the methods directly related to the fields I removed from the YAML, you get a chain of unused methods that were used only from the code I deleted and which we want to remove as well. So it is easy to mix into it some method that was unused already before. I can revert this part if you want, but I do not think it hurts to just remove it.